ui/src/components/Profile.scss

```
@@ -0,0 +1,25 @@
+.profile {
+  & .__section {
+    position: relative;
+    top: -5em;
+    padding-left: 3em;
+    & .__avatar {
+      height: 2em;
+      width: 2em;
+      object-fit: contain;
+      & .__image {
+        box-sizing: border-box;
+        border: 5px solid white;
+        transition: 0.3s;
+        &:hover {
+          transform: scale(1.2);
+        }
+      }
+    }
+  }
+  & .__btn {
+    & .-submit {
+      width: 3rem !important;
+    }
+  }
+}
```

This classes are not following BEM convention. Please try below, see how it expands, and update your html template. Please ping me if you have any questions.

```suggestion
.q-profile {
  .q-profile {
    &__section {
      position: relative;
      top: -5em;
      padding-left: 3em;
      &__avatar {
        height: 2em;
        width: 2em;
        object-fit: contain;
        &__image {
          box-sizing: border-box;
          border: 5px solid white;
          transition: 0.3s;
          &:hover {
            transform: scale(1.2);
          }
        }
      }
    }
    &__btn {
      &--submit {
        width: 3rem !important;
      }
    }
  }
}
```